soda_machine.rb

```diff
@@ -7,12 +9,20 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    sodas.length
   end
 
   def find_soda(soda_brand)
+    sodas.find {|soda| soda.brand == soda_brand}
   end
 
   def sell(soda_brand)
+      if sodas.empty?
```

Comment:
are you checking that the sodas are empty or if that soda brand exists in the sodas collection? Or both? This conditional statement is partially relevant for this situation